dev-python/Rx/Rx-3.1.0.ebuild

```diff
@@ -0,0 +1,13 @@
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 python3_7 )
```

Comment:
Up to 3.9, please.

Reply:
I don’t have a chroot readily available. It seemed to me that, given the choice between spending a few minutes to bump the package three times over the next handful of months as each new Python version becomes stable, versus an hour or two setting up a new install that has 3.8 or 3.9 available, the former was a lot easier. And as far as I could tell it didn’t bother the Python team either; they just went ahead and made 3.7 the standard target anyway without caring that some packages were 3.6-only, so it seemed as if they could do the same with 3.8 and 3.9. If that’s not true, let me know and perhaps I’ll find time to set up a chroot eventually.

Reply:
Unless you’re OK with just this package, but not the whole ecosystem, being tested in 3.8/3.9?

Reply:
The whole point is that this package was last rited because it wasn't timely ported. We should aim higher so that it survives two more months before being last rited again (py3.8 will be the default soon). And yes, it's fine to just test one package.

Reply:
9f3aa9e766ee; unit tests pass on 3.7/3.8/3.9, and I also tested an application I have which uses Rx on both 3.7 and 3.8 (but not 3.9 as it requires other libraries not available there yet).